receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/constant.go

```diff
@@ -23,6 +23,51 @@ const (
 	AttributeECSTaskRevesion = "ecs.task-definition-version"
 	AttributeECSServiceName  = "ecs.service"
 
+	CPUsInVCpu               = 1024
+	BytesInMiB               = 1024 * 1024
 	ContainerMetricsLabelLen = 3
 	TaskMetricsLabelLen      = 6
+
+	TaskPrefix                        = "ecs.task."
+	ContainerPrefix                   = "container."
+	ResourceAttributeServiceNameKey   = "service.name"
+	ResourceAttributeServiceNameValue = "awsecscontainermetrics"
+	MetricResourceType                = "aoc.ecs"
```

Comment:
AWS Observability Collector-> Amazon distribution of OpenTelemetry.

Reply:
FWIW, resource type doesn't exist in OTel protocol but is there right now for metrics since it still seems to use opencensus. So this value will generally be dropped

Reply:
Noted. Even if it gets utilized we want to use "aoc.ecs" to differentiate our OT metrics from ECS backend metrics.

Reply:
I point it out because it will go away - so if you do have an expectation of having it, it won't be there :P But I think if we put the receiver name in something like `telemetry.sdk` than the information should still be preserved, in a way that matches in some sense what our apps send.